nixos/modules/services/misc/jellyfin.nix

```diff
@@ -31,6 +31,22 @@ in
         description = lib.mdDoc "Group under which jellyfin runs.";
       };
 
+      dataDir = mkOption {
+        type = types.path;
+        default = "/var/lib/jellyfin";
+        description = lib.mdDoc ''
+          Jellyfin data directory.
+        '';
+      };
+
+      cacheDir = mkOption {
+        type = types.path;
+        default = "/var/cache/jellyfin";
+        description = lib.mdDoc ''
+          Jellyfin cache data directory.
```

Comment:
The service name is duplicated with the path and the description is rather none descriptive.

Reply:
I think it's ok to keep these explicitly set here with the hardcoded 'jellyfin' in the name. We don't want the defaults to accidentally change as that will break existing setups. Most other services I see (eg postgresql) use the repeated names rather than abstracting it out to another variable